### ISSUE TITLE
Improve text contrast ratios

### ISSUE THREAD

**Issue by AABoyles** [CDCgov/MicrobeTrace-Archive#95]
[WebAIM](https://webaim.org/standards/wcag/checklist)'s most stringent requirement (Guideline 1.4.6) requires a contrast ratio of at least 7:1. The nav bar fails this requirement.

**Comment by AABoyles** [comment]
Improved by 4d5a690af19a824b3c0c771493ac626ade072e6d. Still need to prove its sufficiency.
